<launch>
	<node name="object_publisher"  pkg="pcl_ros" type="pcd_to_pointcloud" args="/home/kalectro/catkin/src/ros_object_recognition/object.pcd" >
		<remap from="/cloud_pcd" to="object_pointcloud" />
	</node>
</launch>
